README.md

```diff
@@ -1,3 +1,78 @@
-Data Migrations
-===============
+# Data Migrations for [Grove][groveco/grove]
 
+This repository represents standalone data migrations for our [main
+application][groveco/grove] that are either in proposal or have been applied to
+the production database. Alongside this `README` file are a number of
+directories indicating the order of the data migrations as they have been
+applied. For example:
+
+```
+-+ README.md -- this document
+ |-+ migrations_0001 -- the first data migration
+ | |- README.md -- What does it do? How do I do it?
+ | |- Pipfile -- Python requirements, if any
+ | |- supporting.py -- referenced in `README.md`
+ | +- some-gnarly.sql -- that I might need to run per `./README.md`
+ +-+ migrations_XXXX -- the XXXX-th migration
+   +- . . . -- you get the picture, I hope.
+```
+
+## The Process
+
+### 1. Create a new branch containing `migrations_XXXX/README.md` that describes the work to be done.
+
+- Include the reasons _why_ the migration is applied, referencing JIRA issues
+  or PRs from [the main repo][grovceco/grove]
+- Include step-by-step instructions for how to apply the migration, referencing
+  support files as necessary.
+- Please _always_ include instructions and support files to reverse-apply the migration.
+
+### 2. Open a Pull Request and solicit code reviews
+
+- Please review the proposed migration up and down and attempt applying it in a safe environment.
+
+### 3. Merge to `master` when approved!
+
+- This signals that the proposed migration slot is "taken" and outstanding PRs
+  should catch up, possibly renumbering their own migrations. [See "Pro
+  Tips"](#pro-tips).
+- Migrations in `master` are "ready to be applied" in a DevOps task. [See
+  "Applying Migrations"](#applying-migrations)
+
+## Applying Migrations
+
+- Create a branch named `applying-migrations---XXXX-YYYY` where `XXXX` is the
```

Comment:
I'm trying to imagine why or when one might run multiple of these manual migrations all at once via one branch? I thought each directory in this repo would designate a single atomic data migration, even if it does more than one set of `creates / updates / deletes / inserts` 

I would think each of these migrations would be atomic in order to keep the complexity low.

Reply:
Valid point. I was trying to cover the bases. I could foresee a situation where multiple manual migrations might get proposed, reviewed, and approved and then run later, perhaps Friday to Monday. It would be much less onerous on an operator to update the repo with which migrations were run in a single merge in that case.

Agreed, the individual migrations should be atomic, regardless of the number of CRUD operations contained therein.